src/root.tsx

```diff
@@ -10,15 +10,63 @@ import {
     Meta,
     Routes,
     Scripts,
-    Title
+    Title,
+    useNavigate
 } from "solid-start";
 
 // eslint-disable-next-line
 import "@mutinywallet/ui/style.css";
 
+import { App as CapacitorApp } from "@capacitor/app";
+import { Capacitor } from "@capacitor/core";
+import { onCleanup } from "solid-js";
```

Comment:
need to merge this import with the one above... tbh odd that `pnpm run format` isn't doing this automatically